src/picongpu/include/simulation_defines/unitless/speciesAttributes.unitless

```diff
@@ -109,7 +109,7 @@ struct Unit<globalCellIdx<T_Type> >
 };
 
 template<>
-struct Unit<chargeState >
+struct Unit<boundElectrons>
 {
   static std::vector<double> get()
   {
```

Comment:
please see [this issue](https://github.com/ComputationalRadiationPhysics/picongpu/issues/707) https://github.com/ComputationalRadiationPhysics/picongpu/issues/707